src/gui/C4StartupScenSelDlg.h

```diff
@@ -474,6 +475,11 @@ class C4StartupScenSelDlg : public C4StartupDlg
 	void OnSelDblClick(class C4GUI::Element *pEl) { DoOK(); }
 	void OnButtonScenario(C4GUI::Control *pEl);
 
+	C4GUI::Edit::InputResult OnSearchBarEnter(C4GUI::Edit *edt, bool fPasting, bool fPastingMore)
+	{
+		UpdateList(); return C4GUI::Edit::IR_Abort;
```

Comment:
Return in next line, please

Reply:
I copy-pasted that code from C4StartupNetDlg.

Reply:
The comments here are not personal criticism :) It doesn't really matter who introduced that code smell, but touching code is a great opportunity to also reduce the technical debt of the project a little bit each time :)